tests/pipelines/test_pipelines_audio_classification.py

```diff
@@ -13,8 +13,10 @@
 # limitations under the License.
 
 import unittest
+from dataclasses import fields
 
 import numpy as np
+from huggingface_hub.inference._generated.types import AudioClassificationOutputElement
```

Comment:
```suggestion
from huggingface_hub import AudioClassificationOutputElement
```

importing from root works as well (usually better than from private module)